src/ClientAppSpec.ts

```diff
@@ -57,9 +56,9 @@ export default describe('ClientApp', () => {
                 ];
 
                 let index = 0;
-                spyOn(ClientApp, '_getQueryString').and.callFake(() => invalidQueryStrings[++index]);
+                spyOn(ClientApp, '_getQueryString').and.callFake(() => invalidQueryStrings[index++] as any);
```

Comment:
Fixed a small bug here.  The index was incrementing ahead of the case.  Ironically, when the index eclipsed the size of the array, it would return undefined and still run the intended test.

Was that too much of a happy accident?  Maybe I should have left the code and a comment,
// intentionally exceeding array bounds to test 'undefined' :joy: